### PR TITLE
[Documentation] Feature: fixed doc blocks

### PR DESCRIPTION
**Pull request by vollborn**
changed doc blocks

### CODE REVIEW

File: doc/Development_Documentation/05_Objects/01_Object_Classes/01_Data_Types/05_Blocks.md [hunk 1]
```diff
@@ -28,9 +28,9 @@ The value of the second input field can be retrieved as follows:
 
 ```php
         $object = DataObject\BlockClass::getById(48);
-        /** @var  $blockData DataObject\ClassDefinition\Data\Block */
+        /** @var DataObject\ClassDefinition\Data\Block $blockData */
```
Comment (blankse) [inline doc/Development_Documentation/05_Objects/01_Object_Classes/01_Data_Types/05_Blocks.md:31]:
```suggestion
        /** @var DataObject\ClassDefinition\Data\Block $blockItems */
```

File: doc/Development_Documentation/05_Objects/01_Object_Classes/01_Data_Types/05_Blocks.md [hunk 2]
```diff
@@ -49,9 +49,9 @@ Get Values of Localized Block Entries:
...
```
Comment (blankse) [inline doc/Development_Documentation/05_Objects/01_Object_Classes/01_Data_Types/05_Blocks.md:52]:
```suggestion
        /** @var DataObject\ClassDefinition\Data\Block $blockItems */
```

**Comment by blankse** [comment]
@vollborn I think we should remove it complete. The class types (and partly the variable name) are wrong and the return doc from the methods are correct now.

**Comment by brusch** [comment]
Thanks!

**Pull request merged**